### ISSUE TITLE
Additional calendar

### ISSUE THREAD

**Issue by kristenruff** [givingforum/givingforum#209]
Hi all,
We keep a "community calendar" on our website where we publicize member events that PNY is not co-sponsoring. It's a pretty pathetic page - just a bunch of text - so we're interested in giving it a face lift. We'd like to use the event content type but keep these events separate from our main calendar. I was thinking something along the lines of adding events but tagging them as "community calendar" so they don't sync to SF and don't show up on the main calendar list view but we could pull them into views. 

I haven't thought this out completely but wanted to see if any other RAs might want to do something like this. 

- Kristen / PNY

**Comment by allenlaurie** [comment]
CCP also has an “Other Events” page: https://www.ctphilanthropy.org/other-organization-events

We’d be interested in a “community calendar.”

[cid:image001.jpg@01D0F091.F2466C10]
Laurie Allen | Communications Manager
Connecticut Council for Philanthropy
860-525-5585 | lallen@CTphilanthropy.org<mailto:lallen@CTphilanthropy.org>
221 Main Street, Hartford, CT  06106
[Conference Signature 2016 #2]<https://www.ctphilanthropy.org/events/2016-ccp-philanthropy-luncheon-conference>

**Comment by MarilouJones** [comment]
We promote “other” events in our e-newsletters but not on our website. Our stats showed that page got almost no views, so we got rid of it.

Marilou

**Comment by vrozansky** [comment]
Good idea. I think to avoid any overlap issues with regular events, we can create a new content type for Regional Events (Ex: regional_event). Then, we can add all the necessary fields that will resemble our events (please feel free to list them here including any new fields) and generate this calendar pretty easily. Open to any other suggestions. Thank you.

**Comment by vrozansky** [comment]
Please chime in by March 18th on this ticket so we can determine if there's a joint task here and properly estimate it. Thank you.

**Comment by PhilOhio** [comment]
POH is not interested at this time.

**Comment by mtrimbach** [comment]
CNJG isn't interested at this time.

**Comment by cjoien** [comment]
Not something MCF will be pursuing at the moment.

**Comment by microveldt** [comment]
PNW is not interested at this time

**Comment by EmilyBaron** [comment]
gwp is not interested at this time

**Comment by sgeis** [comment]
Indiana is not interested at this time.

**Comment by RebekahSeder** [comment]
WRAG is not interested. We promote other events on our blog.

**Comment by canheier** [comment]
SCG - yes
